TTS/config/shared_configs.py

```diff
@@ -205,11 +205,20 @@ class BaseTrainingConfig(Coqpit):
             the end of the evaluation. Default to ```False```.
         print_step (int):
             Number of steps required to print the next training log.
-        tb_plot_step (int):
+        log_dashboard (str): "tensorboard" or "wandb"
+            Set the experiment tracking tool
+        plot_step (int):
             Number of steps required to log training on Tensorboard.
-        tb_model_param_stats (bool):
+        model_param_stats (bool):
             Enable / Disable logging internal model stats for model diagnostic. It might be useful for model debugging.
             Defaults to ```False```.
+            wandb_disabled: bool = False
+        project_name (str):
+            Name of the project. Defaults to config.model
+        wandb_entity (str):
+            Name of W&B entity/team. Enables collaboration across a team or org.
+        wandb_log_model_step (int):
```

Comment:
I guess we don't need this docstring anymore .

Reply:
Ahh yes. Thanks for pointing this out. I'll fix it

Reply:
`wandb_entity` is sort of useful as we discussed above. But yeah, I've changed `wandb_log_model_step` to `log_model_step`